packages/Toggle/src/Toggle.tsx

```diff
@@ -40,19 +51,26 @@ const focusStateStyle = css`
   }
 `;
 
-const getContainerStyles = ({ size, disabled }) => {
-  const sizeStyle = {
-    default: css`
+interface StateForStyle {
```

Comment:
I don't love the name `StateForStyle`. Any suggestions on a better one?

Reply:
Meh, it works for me. Can't think of anything better either.

Reply:
Possibly `StatefulStyleProps`? Not sure that's better?